src/include/network/lan/address.rb

```diff
@@ -1355,6 +1355,19 @@ def AddressDialog
         Ops.set(@settings, "VLAN_ID", Builtins.tointeger(LanItems.vlan_id))
       end
 
+      if LanItems.type == "br"
+        ports = LanItems.bridge_ports.split(" ").select { |p| old_bridge_config?(p) }
+
+        unless ports.empty?
+          message = format(_("The bridge ports listed below are configured in the old way.\n\n" \
+                            "Bridge ports: %s\n\n" \
+                            "Do you want to adapt them now?"), ports.join(", "))
+          if Popup.YesNoHeadline(Label.WarningMsg, message)
+            ports.each { |p| configure_as_bridge_port(p) }
```

Comment:
We are just adapting it when the bridge is edited as we can see in this gif:

![ask_for_adapt](https://cloud.githubusercontent.com/assets/7056681/18754130/79908456-80df-11e6-91ea-c864201f606c.gif)


We could also ask for adapt the configuration when the enslaved interface is edited.